content/en/docs/contribute/advanced.md

```diff
@@ -59,6 +57,19 @@ reviewed is usually small. These queries specifically exclude localization PRs,
   Determine whether any additional changes or updates need to be made for the PR to be merged. If you think the PR is ready to be merged, comment `/approve`.
 - [Not against master](https://github.com/kubernetes/website/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+-label%3Ado-not-merge+label%3Alanguage%2Fen+-base%3Amaster): If it's against a `dev-` branch, it's for an upcoming release. Make sure the [release meister](https://github.com/kubernetes/sig-release/tree/master/release-team) knows about it by adding a comment with `/assign @<meister's_github-username>`. If it's against an old branch, help the PR author figure out whether it's targeted against the best branch.
 
+### When to close Pull Requests
+
+Reviews and approvals are one tool to keep our PR queue short and current. Another tool is closure. 
+
+- Close any PR where the CLA hasn’t been signed for two weeks. 
+PR authors can reopen the PR after signing the CLA, so this is a low-risk way to make sure nothing gets merged without a signed CLA.
+
+- Close any PR where the author has not responded to comments or feedback in 2 or more weeks.
+
+Don't be afraid to close pull requests. Contributors can easily reopen and resume works in progress. Oftentimes a closure notice is what spurs an author to resume and finish their contribution.
+
+To close a pull request, leave a `/close` comment on the PR.
```

Comment:
In case that we need the PR can we become a co-author for the PR?

Reply:
> In case that we need the PR can we become a co-author for the PR?

If you're asking whether or not you can "take over" the PR by pulling the commit locally, working on it, and pushing up a different PR, the answer is yes, of course, but it is best to coordinate with the original author to ensure they have actually abandoned their efforts.

Reply:
@irvifa Sure. Would you agree, though, that co-authoring is out of scope for this particular PR?

Reply:
Yep, I agree for that. Thanks 🙁 Another case if we can't really ping the previous author what can be done?

Reply:
@irvifa In that case cherry-picking seems like a reasonable choice.

Reply:
*Aside* In a couple of cases I have adopted commits from another author's PR (and communicated about this). I definitely wouldn't adopt commits from an author who had not yet signed the CLA, because the subsequent pull request would be on shaky ground in terms of licensing.